### ISSUE TITLE
what's the difference between uni3d_g and uni3d_g with `fire` symbol?

### ISSUE THREAD

**Issue by auniquesun** [baaivision/Uni3D#21]
Thanks for sharing the paper, code and weights. It's a great work!

One point confuses me is that in the model zoo, you have two uni3d_g models pre-trained with the **Ensemble** datasets, as the red box highlights below. 

![1719708639458](https://github.com/baaivision/Uni3D/assets/25259958/aa5822c1-87c5-480a-8c2d-f9c9813c3ca0)

But their zero-shot accuracy is quite different, uni3d_g with the `fire` symbol is much better than uni3d_g, so what's the difference between these two models? 

Looking forward to your reply. Thanks.

**Comment by auniquesun** [comment]
@mabaorui @encounter1997 @Unrealluver @WXinlong  It would be appreciated if you can provide some insights.
